src/components/dev-hub/card-list.js

```diff
@@ -41,71 +41,76 @@ const getThumbnailUrl = media => {
         : media.thumbnailUrl;
 };
 
-// TODO: Update this component to take one array of items and map to cards based
-// on the item type
+const sortCardsByDate = contentList =>
+    contentList.sort((a, b) => {
+        if (new Date(a.publishDate) > new Date(b.publishDate)) return -1;
+        if (new Date(a.publishDate) < new Date(b.publishDate)) return 1;
+        return 0;
+    });
+
+const showArticles = article => (
```

Comment:
nit, but since these expect a single media object, can we rename to be singular? (same for videos, podcasts)
```suggestion
const showArticle = article => (
```